nuxeo-services/nuxeo-platform-web-common/src/main/java/org/nuxeo/ecm/platform/ui/web/auth/NuxeoAuthenticationFilter.java

```diff
@@ -873,6 +870,10 @@ protected boolean handleLogout(ServletRequest request, ServletResponse response,
         } catch (LoginException e) {
             log.error("Unable to logout " + e.getMessage());
         }
+
+        // invalidate Session !
+        service.invalidateSession(request);
+
```

Comment:
could you please explain why you had to move this invalidation? this class is pretty sensitive to changes and this could impact other login plugins that are not tested by default: i wonder why this would be needed only for shibboleth